addons/web/static/src/legacy/js/fields/basic_fields.js

```diff
@@ -571,11 +572,74 @@ var FieldChar = InputField.extend(TranslatableFieldMixin, {
     tagName: 'span',
     supportedFieldTypes: ['char'],
     isQuickEditable: true,
+    dynamicPlaceholderOptions: {
```

Comment:
I think the `this` space is already crowded enough to try to avoid adding new stuff to it if it's not really needed. I understand it was done in this way to allow for overriding but the other definition of `dynamicPlaceholderOptions` uses all the exact same values as the default one so I'm confused.

Reply:
True, Maybe it's too much of a preemptive optimization.
We can bring back the options as a const in the function `openDynamicPlaceholder` to avoid adding stuff in the this space.

WYT ?

Reply:
Yes I think I would prefer it to be back as const for now if you don't mind.

Reply:
Will try this in another commit

Reply:
see c73b30a5ff591f58284669a27576cc198c902c2f